packages/editor-ui/src/modules/communityNodes.ts

```diff
@@ -0,0 +1,29 @@
+import { getCommunityPackageCount } from '@/api/settings';
+import { ICommunityNodesState, IRootState } from '@/Interface';
+import Vue from 'vue';
+import { ActionContext, Module } from 'vuex';
+
+const module: Module<ICommunityNodesState, IRootState> = {
+	namespaced: true,
+	state: {
+		availablePackageCount: 0,
+	},
+	mutations: {
+		setPackageCount: (state: ICommunityNodesState, count: number) => {
```

Comment:
nitpick - might be clearer `setAvailablePackageCount`

Reply:
Agree.